firmware/mcal/cli/periph/gpio.h

```diff
@@ -35,8 +36,7 @@ class DigitalOutput : public shared::periph::DigitalOutput {
     DigitalOutput(std::string name) : name_(name) {}
 
     void Set(bool value) override {
-        std::cout << "Setting DigitalOutput Channel " << name_ << " to "
-                  << (value ? "true" : "false") << std::endl;
+        std::cout << std::format("Setting DigitalOutput Channel {} to {}", name_, (value ? "true": "false") ) << std::endl;
```

Comment:
Check the cpp format string spec. Do we need the ternary operator or is there a simpler way to print a string version of a Boolean